packages/migrate/commonsChunkPlugin/__snapshots__/commonsChunkPlugin.test.js.snap

```diff
@@ -0,0 +1,303 @@
+"module.exports = {
+    entry: {
+		main: './src/index.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                main: {
+                    name: 'main',
+                    chunks: 'initial',
+                    enforce: true,
+
+                    test: module => {
+                        if (module.getChunks().some(chunk => chunk.name === 'main'))
+                            return true;
+
+                        const fn = ({ resource }) => {
+                            // some code
+                            return /node_modules/.test(resource);
+                        };
+
+                        return fn(module);
+                    }
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-6c" data 1`] = `
+"module.exports = {
+    entry: {
+		main: './src/index.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                main: {
+                    name: 'main',
+                    chunks: 'initial',
+                    enforce: true,
+
+                    test: module => {
+                        if (module.getChunks().some(chunk => chunk.name === 'main'))
+                            return true;
+
+                        const fn = function ({ resource }) {
+                            // some code
+                            return /node_modules/.test(resource);
+                        };
+
+                        return fn(module);
+                    }
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-6d" data 1`] = `
+"module.exports = {
+    entry: {
+		main: './src/index.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                main: {
+                    name: 'main',
+                    chunks: 'initial',
+                    enforce: true,
+
+                    test: module => {
+                        if (module.getChunks().some(chunk => chunk.name === 'main'))
+                            return true;
+
+                        const fn = function ({ resource }) {
+                            if (foo) {
+                                return /node_modulesfoo/.test(resource);
+                            } else {
+                                return /node_modulesbaz/.test(resource);
+                            }
+                        };
+
+                        return fn(module);
+                    }
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-7" data 1`] = `
+"module.exports = {
+    entry: {
+		main: './src/index.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            minSize: 3000,
```

Comment:
This need to be moved into `cacheGroups.main`

Reply:
Fixed.